rocketchat/post_install.md

```diff
@@ -0,0 +1,32 @@
+## Rocket chat
+
+### External access
+
+By default external access to the Rocket chat isn't available. This is easily changed by applying the following YAML to apply ingress via traefik to your cluster with `kubectl apply -f rocketchat-ingress.yaml` (or whatever you call the file containing the contents below):
+
+```
+apiVersion: extensions/v1beta1
```

Comment:
Civo Kubernetes uses K3s 1.20, in 1.22+, this ingress type won't be available on Kubernetes anymore. Refer to this [article](https://www.civo.com/learn/migrating-your-ingresses-in-k3s-1-20).

Reply:
yup till this will work and ill update it after 1.22+, you can also contribute if you want right!